temporaltest/server.go

```diff
@@ -39,10 +40,43 @@ func (ts *TestServer) fatal(err error) {
 }
 
 // Worker registers and starts a Temporal worker on the specified task queue.
+// WorkflowPanicPolicy is set to worker.FailWorkflow
 func (ts *TestServer) Worker(taskQueue string, registerFunc func(registry worker.Registry)) worker.Worker {
```

Comment:
Hmm just thinking out loud, we should probably deprecate this and rename to `NewWorker` before the first release. That would be more consistent with `NewClientWithOptions` and `NewWorkerWithOptions`.

Reply:
I can work on a followup PR that changes this. Should it be a breaking change, or should it support both methods?

Reply:
We can just mark as deprecated for now (no breaking change), and then remove it before the first tagged release.